benchmarks/encryption.py

```diff
@@ -0,0 +1,330 @@
+from abc import ABC
+from typing import IO
+from typing import Iterable
+from os import urandom
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+from base64 import b64encode
+
+
```

Comment:
Nitpick: only one newline between import blocks. I'll add an import style linter for the project so that we're consistent with webapp and cloudserver.

Reply:
The linter is now added, so if you rebase or merge your branch onto master then Travis will run isort as part of the CI.

Reply:
Ok awesome! I think it's about time for me to add an auto pep8 linter to my text editor 😅